.github/workflows/continuous-integration-workflow.yml

```
@@ -0,0 +1,65 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+      - develop
+  pull_request:
+    branches:
+      - master
+      - develop
```

```suggestion
on: [push, pull_request]
```
Let's be consistent with the OSX check here.